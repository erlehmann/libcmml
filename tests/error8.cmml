<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<head>
<title>
Australia Advances - Hidden Galaxies
</title>
<meta name="DC.audience" content="General"/>
<meta name="DC.author" content="CSIRO Publishing"/>
<meta name="DC.date.created" content="2001/07/30"/>
<meta name="DC.date.issued" content="2001/07/30"/>
<meta name="DC.description" content="Australia Advances Video - Hidden Galaxies"/>
<meta name="DC.format" content="video"/>
<meta name="DC.language" content="English"/>
<meta name="DC.publisher" content="CSIRO Australia"/>
<meta name="DC.rights" content="http.//www.csiro.au/rights"/>
<meta name="DC.subject" content="galaxies, hidden, telescope, gas, astronomers, mikly, way, australia, oz, advances, csiro, videos, movies, stories, research, science, technology, innovation, achievements"/>
<meta name="DC.title" content="Australia Advances - Hidden Galaxies"/>
<meta name="DC.title.alt" content="Australia Advances - Hidden Galaxies"/>
<meta name="keywords" content="galaxies, hidden, telescope, gas, astronomers, mikly, way, australia, oz, advances, csiro, videos, movies, stories, research, science, technology, innovation, achievements"/>
</head>

<clip id="intro" start="npt:0">
<a href="http://www.csiro.au/promos/ozadvances/">
http://www.csiro.au/promos/ozadvances/</a>
<img src="intro.jpg"/>
<desc>
Introduction to 'Hidden Galaxies - Australia Advances video'.
</desc>
</clip>

<head>
<title>
Australia Advances - Hidden Galaxies
</title>
</head>

<clip id="startspeech" start="npt:9">
<meta name="KEYWORDS" content="Austronomy, Radio-Telescope, Galaxies"/>
<a href="http://www.aao.gov.au/galaxies.anx#radio">
Related video on Detection of Galaxies</a>
<img src="findinggalaxies.jpg"/>
<desc>
What's out there? It's the question that everyone wants
          answered. And now, with the most powerful telescope of it's
          kind in the world, Australian astronomers may be about to
          find out in their search for hidden galaxies. Even with
          a radio telescope as powerful as the one at Parkes,
          astronomers can take a long time to search the skies. Any
          galaxies beyond our own are faint and give out little
          light. Some are hidden by the stars and the dust of the
          Milky Way. Astronomers find hidden galaxies by looking for
          radio waves that come from cool hydrogene gas, the stuff
          that stars are made from. But the problem in the past has
          been that the radio waves are so weak that it takes a long
          time to find them.
      
</desc>
</clip>

<clip id="parkes" start="npt:64">
<meta name="KEYWORDS" content="Austronomy, Radio-Telescope, Parkes"/>
<a href="http://www.parkes.atnf.csiro.au/">
More information on the Parkes Radio Telescope</a>
<img src="parkes.jpg"/>
<desc>
Visiting Parkes:
         open to the public Mo-Fr, 8:30 - 16:15, except Christmas and Boxing Days
         Admission: free
         Audio-Visual display: $3.00 adults, $2.00 concession, $10.00 family
         Address:
         Australian Telescope National Facility
         Parkes Observatory
         Alectown
         PO Box 276
         Parkes
         NSW 2870
         Australia
         Situated:
         25 km North of Parkes
         365 km West of Sydney
         6 km off Newell Highway running between Parkes and Dubbo
         For reservations call:
         +61-2-68611700
      
</desc>
</clip>

<clip id="magellanic" start="npt:102">
<img src="magellanic.jpg"/>
<desc>
Dr. Lister Staveley-Smith from CSIRO predicts: "Magellanic
      clouds will collide with our own galaxy in about 1 billion
      years."
      
</desc>
</clip>

<clip id="newgalaxies" start="npt:123">
<meta name="KEYWORDS" content="Astronomy, new galaxies"/>
<a href="http://www.aao.gov.au/galaxies.mpeg">
Related video on Detection of Galaxies</a>
<img src="newgalaxies.jpg"/>
<desc>
100 new galaxies and 220 new pulsars detected.
      
</desc>
</clip>

<clip id="extro" start="npt:145">
<a href="http://www.csiro.au/promos/ozadvances/">
Other Australia Advances videos</a>
<img src="extro.jpg"/>
<desc>
Ending of 'Hidden Galaxies - Australia Advances video'.
      
</desc>
</clip>

</cmml>
