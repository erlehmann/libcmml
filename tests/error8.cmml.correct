<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<head>
<title>
Australia Advances - Hidden Galaxies
</title>
<meta name="DC.audience" content="General"/>
<meta name="DC.author" content="CSIRO Publishing"/>
<meta name="DC.date.created" content="2001/07/30"/>
<meta name="DC.date.issued" content="2001/07/30"/>
<meta name="DC.description" content="Australia Advances Video - Hidden Galaxies"/>
<meta name="DC.format" content="video"/>
<meta name="DC.language" content="English"/>
<meta name="DC.publisher" content="CSIRO Australia"/>
<meta name="DC.rights" content="http.//www.csiro.au/rights"/>
<meta name="DC.subject" content="galaxies, hidden, telescope, gas, astronomers, mikly, way, australia, oz, advances, csiro, videos, movies, stories, research, science, technology, innovation, achievements"/>
<meta name="DC.title" content="Australia Advances - Hidden Galaxies"/>
<meta name="DC.title.alt" content="Australia Advances - Hidden Galaxies"/>
<meta name="keywords" content="galaxies, hidden, telescope, gas, astronomers, mikly, way, australia, oz, advances, csiro, videos, movies, stories, research, science, technology, innovation, achievements"/>
</head>

<clip id="intro" track="default" start="npt:0.000">
<a href="http://www.csiro.au/promos/ozadvances/">
http://www.csiro.au/promos/ozadvances/</a>
<img src="intro.jpg"/>
<desc>
Introduction to 'Hidden Galaxies - Australia Advances video'.
</desc>
</clip>

