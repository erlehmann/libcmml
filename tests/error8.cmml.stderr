error8.cmml:34:0: head tag after clip tag not allowed
